GLAA.Domain/Models/Address.cs

```diff
@@ -14,12 +15,14 @@ public class Address : IId
 
         public string Town { get; set; }
 
-        public string County { get; set; }
-
         public string Postcode { get; set; }
 
-        public string Country { get; set; }
+        public int? CountryId { get; set; }
+        [ForeignKey(nameof(CountryId))]
+        public virtual Country Country { get; set; }
 
-        public bool NonUK { get; set; }            
```

Comment:
With the removal of the NonUK flag, does this force the user to enter data under the guise of mandatory which may then be set to non mandatory when they select the country? I assume this is all agreed wireframed etc? Reason I mention is is that DD has quite vocal on having 'Non UK' as a flag before filling in the address info.

Reply:
It's removed in Adam's wireframes, which I assume DD has seen? There is still a concept of a UK/Non UK address although the logic is driven by the country dropdown. The flow is slightly different as a user can say that they have an address outside the UK and still enter "UK - England" but the validation (when it's finished) will force them to fill in the appropriate fields.